src/components/Profile/Profile.css

```diff
@@ -0,0 +1,106 @@
+* {
+    box-sizing: border-box;
+  }
+  .container {
+    margin: 20px;
+    padding: 10px;
+    border-radius: 2px;
+    background-color: #ffffff;
+    box-shadow: 5px 10px 8px 10px #888888;
+    min-height: 300px;
+  }
+  
+  .left {
+    padding: 20px;
+    float: left;
+    text-align: center;
+    width: 30%; /* The width is 30%, by default */
+  }
+  
+  .profile-image {
+    width: 200px;
```

Comment:
These values are fixed in pixels which will not make it responsive for bigger screens or desktops. So, try to change it to % or rem.